docs/design/README.md

```diff
@@ -2127,38 +2123,42 @@ package Geometry library "Shapes" api;
 
 Parts of this declaration may be omitted:
 
--   If the package name is omitted, as in `package library "Main" api;`, the
-    file contributes to the default package. No other package may import from
-    the default package.
+-   If the package keyword is not specified, as in `library "Main" api;`, the
+    file contributes to the main package. No other package may import from the
+    main package.
```

Comment:
What do you think about dropping this form entirely given that we have a name for the default package now?

I'm a bit torn, I do see the niceness of being able to write a handful of files w/o even picking a package name. But it's also another rule. Interested in what others think.

Reply:
I'm torn too. So maybe we should choose whichever one we expect to learn more from? I don't think we'll be able to learn much from keeping this rule: the cost of an extra rule is probably too diffuse to notice, and the benefit will probably be mostly invisible to us, because it'll tend to get used in things like prototypes and experiments that are usually not published or shared where we can see them. Conversely, if we omit this rule, I think there's a decent chance that we'll hear complaints if that omission is a problem.

Reply:
This would be the only place in source files where the name `Main` would be usable as a package name, which seems a little inconsistent, and if you can write `package Main library "foo";` I think people would ask why you can't write `import Main library "foo";`. But we could just allow both, and say that it's only the default library of the `Main` package that you can't import, because it has no `api` file. I think I'm happy here either way; I don't think there's a huge amount of value in preventing the main package from being imported from some other package (but it would definitely be weird if there are cycles in the package dependency graph at all).

Reply:
The main concern I've heard is that we don't want to violate the [one-way principle](https://github.com/carbon-language/carbon-lang/blob/trunk/docs/project/principles/one_way.md). If we want a short package declaration for common and simple cases, and want largely consistent behavior, then I think we probably have to accept only the shortest consistent form.